packages/analytics/dashboard-renderer/src/components/DashboardRenderer.cy.ts

```diff
@@ -99,23 +98,14 @@ describe('<DashboardRenderer />', () => {
         },
       }
 
-      console.log('Resolving config')
-      return Promise.resolve(config)
-    }
-
-    // @ts-ignore: TS doesn't infer things correctly.  NoInfer may help.
-    const evaluateFeatureFlagFn: AnalyticsBridge['evaluateFeatureFlagFn'] = (key) => {
```

Comment:
I think we still want this stub; it should just return true in all circumstances.